pages/careers/_open-positions.html

```diff
@@ -16,7 +16,7 @@ <h4>Your Ideal Background</h4>
       <li>You know how to write code across the stack ("Dev").</li>
       <li>You have experience running production software ("Ops").</li>
       <li>You have a strong background in software engineering (or are working hard on it!).</li>
-      <li>Bonus points for outstanding sense of humor.</li>
+      <li>Bonus points for sense of humor, empathetic communication and curiosity.</li>
```

Comment:
Love it. Minor grammar tweak: "Bonus points for a sense of humor, empathy, and curiosity."